pkg/psalabelsyncer/podsecurity_label_sync_controller.go

```diff
@@ -301,11 +383,18 @@ func (c *PodSecurityAdmissionLabelSynchronizationController) checkNSControlled(n
 
 func checkNSControlled(ns *corev1.Namespace) bool {
```

Comment:
i just realized this, to stay go idiomatic (and easier to understand), call this `isNSControlled` and add a godoc (despite not being public this is a very central place of logic).